java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Add SUSE Liberty Linux support for RHEL9 and CentOS Stream 9 based clients
```

Comment:
"CentOS Stream 9" is not supported by SUSE Manager. We should not name it in a changelog